app/containers/AskQuestion/renderTextInput.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import messages from './messages';
+
+function renderTextInput({
```

Comment:
There is already renderTextInput function implemented for other components. They seem very similar. Can we reuse? How about we create reusable fields and validators for forms?